application/config/features.php

```diff
@@ -14,11 +14,12 @@
 
 	// Post views
 	'views' => [
-		'map' => TRUE,
+		'map' => FALSE,
```

Comment:
Whats up with this? map views should be enabled by default - otherwise OSS users are going to find they have no map view.